python/pyspark/ml/torch/tests/test_distributor.py

```diff
@@ -328,11 +328,11 @@ def test_get_num_tasks_locally(self) -> None:
 
     def test_get_gpus_owned_local(self) -> None:
         addresses = ["0", "1", "2"]
-        self.assertEqual(get_gpus_owned(self.spark), addresses)
+        self.assertEqual(_get_gpus_owned(self.spark), addresses)
 
         env_vars = {"CUDA_VISIBLE_DEVICES": "3,4,5"}
         self.setup_env_vars(env_vars)
-        self.assertEqual(get_gpus_owned(self.spark), ["3", "4", "5"])
+        self.assertEqual(_get_gpus_owned(self.spark), ["3", "4", "5"])
         self.delete_env_vars(env_vars)
```

Comment:
Let's add spark connect mode tests for local_training with GPU on the following cases:
 `spark.master=local` and `spark.master=local-cluster`